glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -25,21 +27,25 @@ open class GleanInternalAPI internal constructor () {
     // `internal` so this can be modified for testing
     internal var handle: MetricHandle = 0L
 
+    internal lateinit var configuration: Configuration
+
+    private lateinit var gleanDataDir: File
+
     /**
      * Initialize glean.
```

Comment:
nit: since we're here, let's copy over the docstring from [here](https://github.com/mozilla-mobile/android-components/blob/a806229c3b6e721823a82e67e9146cc1b6f3b730/components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt#L65-L78)